src/home/homectl.c

```diff
@@ -2485,12 +2485,6 @@ static int create_interactively(void) {
 
         any_key_to_proceed();
 
-        r = acquire_bus(&bus);
```

Comment:
hmm, but this means the `_cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;` line above can be dropped too, no?